libdrgn/program.c

```diff
@@ -529,6 +550,14 @@ struct drgn_error *drgn_program_get_dwfl(struct drgn_program *prog, Dwfl **ret)
 	return NULL;
 }
 
+const struct drgn_language *drgn_program_get_lang(struct drgn_program *prog) {
+	if (prog->lang) {
+		return prog->lang;
+	}
+	// TODO handle fallback better.
```

Comment:
Same as `drgn_dwarf_to_language()`, I think for now falling back to C is fine, and in the future we can have a default unknown language.